scripts/scil_clean_qbx_clusters.py

```diff
@@ -0,0 +1,285 @@
+    p.add_argument('--background_opacity', type=float, default=0.1,
+                   help='Opacity of the background streamlines.'
+                   'Keep low between 0 and 0.5 [%(default)s].')
+    p.add_argument('--background_linewidth', type=float, default=1,
+                   help='linewidth of the background streamlines [%(default)s].')
+    p.add_argument('--clusters_linewidth', type=float, default=1,
+                   help='linewidth of the background streamlines [%(default)s].')
+
+    add_reference_arg(p)
+    add_overwrite_arg(p)
+    add_verbose_arg(p)
+
+    return p
+
+
+def get_length_tuple(elem):
+    return len(elem[0])
+
+
+def main():
+    # Callback required for FURY
+    def keypress_callback(obj, _):
+        key = obj.GetKeySym()
+        key = key.lower()
+        nonlocal clusters_linewidth, background_linewidth
+        nonlocal curr_streamlines_actor, concat_streamlines_actor, show_curr_actor
+        iterator = len(accepted_streamlines) + len(rejected_streamlines)
+        renwin = interactor_style.GetInteractor().GetRenderWindow()
+        renderer = interactor_style.GetCurrentRenderer()
+
+        if key == 'c' and iterator < len(sft_accepted_on_size):
+            if show_curr_actor:
+                renderer.rm(concat_streamlines_actor)
+                renwin.Render()
+                show_curr_actor = False
+                logging.info('Streamlines rendering OFF')
+            else:
+                renderer.add(concat_streamlines_actor)
+                renderer.rm(curr_streamlines_actor)
+                renderer.add(curr_streamlines_actor)
+                renwin.Render()
+                show_curr_actor = True
+                logging.info('Streamlines rendering ON')
+            return
+
+        if key == 'q':
+            show_manager.exit()
+            if iterator < len(sft_accepted_on_size):
+                logging.warning(
+                    'Early exit, everything remaining to be rejected.')
+            return
+
+        if key in ['a', 'r'] and iterator < len(sft_accepted_on_size):
+            if key == 'a':
+                accepted_streamlines.append(iterator)
+                choices.append('a')
+                logging.info('Accepted file %s',
+                             filename_accepted_on_size[iterator])
+            elif key == 'r':
+                rejected_streamlines.append(iterator)
+                choices.append('r')
+                logging.info('Rejected file %s',
+                             filename_accepted_on_size[iterator])
+            iterator += 1
+
+        if key == 'z':
+            if iterator > 0:
+                last_choice = choices.pop()
+                if last_choice == 'r':
+                    rejected_streamlines.pop()
+                else:
+                    accepted_streamlines.pop()
+                logging.info('Rewind on step.')
+
+                iterator -= 1
+                renderer.rm(curr_streamlines_actor)
```

Comment:
This need not to be done here, it's done in the following if condition on line 136